### PR TITLE
Add profile type enum

### PR DESCRIPTION
**Pull request by Gill-7**
Add the profile enum type to differentiate between pet sitter and pet owner.

### CODE REVIEW

**Comment by ezeilo-su** [comment]
Hey @Gill-7, please use more appropriate PR titles.

**Review by ezeilo-su — COMMENTED** [review]
Good progress with this one. Please remember to link your tickets in your future PRs. See [guide](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)

File: server/models/Profile.js [hunk 1]
```diff
@@ -4,7 +4,12 @@ const profileSchema = new mongoose.Schema({
   userId: {
     type: mongoose.Schema.Types.ObjectId,
     required: true,
-    ref: 'User'
+    ref: 'User',
+  },
+  accountType: {
+    type: String,
+    default: "",
```
Comment (rajivtitus) [inline server/models/Profile.js:11]: We can make `pet_owner` as the default type 👍🏽

**Review by rajivtitus — APPROVED** [review]
Good work! Just a minor comment to address before merging 😄

**Pull request merged**